dev-db/gigabase/gigabase-3.83-r2.ebuild

```diff
@@ -11,17 +11,23 @@ SRC_URI="mirror://sourceforge/gigabase/${P}.tar.gz"
 
 LICENSE="MIT"
 SLOT="0"
-KEYWORDS="amd64 x86"
+KEYWORDS="~amd64 ~x86"
 IUSE="doc static-libs"
 
 DEPEND="doc? ( app-doc/doxygen )"
 RDEPEND=""
 
 S="${WORKDIR}/${PN}"
 
-src_prepare() {
-	epatch "${FILESDIR}/${P}-fix-dereferencing.patch"
-}
+PATCHES=(
+	"${FILESDIR}/${P}-fix-dereferencing.patch"
+	"${FILESDIR}/${P}-cpp14.patch" # fix #594550
+)
+
+HTML_DOCS=(
+	GigaBASE.htm
+	docs/html/
+)
 
 src_configure() {
 	mf="${S}/Makefile"
```

Comment:
also, change `src_compile` with a proper if clause instead of these ugly subshells